charts/coms/templates/secret.yaml

```diff
@@ -1,10 +1,18 @@
 {{- $bPassword := (randAlphaNum 32) | b64enc }}
 {{- $bUsername := (randAlphaNum 32) | b64enc }}
+{{- $kPassword := (randAlphaNum 32) | b64enc }}
+{{- $kUsername := (randAlphaNum 32) | b64enc }}
+{{- $oPassword := (randAlphaNum 32) | b64enc }}
+{{- $oUsername := (randAlphaNum 32) | b64enc }}
```

Comment:
We can probably drop these as I do not believe there are any circumstances where we would ever want to use this.